hindi/actor4/ompuri.md

```diff
@@ -0,0 +1 @@
+ओम पुरी हिन्दी फिल्मों के एक प्रसिद्ध अभिनेता हैं। इनका जन्म १८ अक्टूबर 1950 में अम्बाला नगर में हुआ। इन्होने ब्रिटिश तथा अमेरिकी सिनेमा में भी योगदान किया है। ये
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.